optaweb-vehicle-routing-backend/src/main/java/org/optaweb/vehiclerouting/plugin/planner/change/RemoveVehicle.java

```diff
@@ -33,39 +33,34 @@ public RemoveVehicle(PlanningVehicle removedVehicle) {
     }
 
     @Override
-    public void doChange(ScoreDirector<VehicleRoutingSolution> scoreDirector) {
-        VehicleRoutingSolution workingSolution = scoreDirector.getWorkingSolution();
-
+    public void doChange(VehicleRoutingSolution workingSolution, ProblemChangeDirector problemChangeDirector) {
         // Look up a working copy of the vehicle
-        PlanningVehicle workingVehicle = scoreDirector.lookUpWorkingObject(removedVehicle);
+        PlanningVehicle workingVehicle = problemChangeDirector.lookUpWorkingObject(removedVehicle);
         if (workingVehicle == null) {
             throw new IllegalStateException("Can't look up a working copy of " + removedVehicle);
         }
```

Comment:
@rsynek This is obsolete as in the capacity change, right?

Reply:
The check was probably redundant from the beginning. Now's a good time to get rid of it.